modules/permutiveRtdProvider.js

```diff
@@ -125,6 +127,43 @@ function getDefaultBidderFn (bidder) {
       }
 
       return bid
+    },
+    pubmatic: function(bid, data, acEnabled) {
+      let segments = [];
+      var existingData = config.getBidderConfig(); // read existing bidder level configs if any
```

Comment:
Can we handle merging of data in PubMaticAdapter and here can we just read data from permutative and send it to bidder.

Reply:
its kept here, since its permutive specific.